.goreleaser.yaml

```diff
@@ -8,9 +8,16 @@ builds:
     env:
       - CGO_ENABLED=0
     goos:
+      - darwin
       - linux
       - windows
-      - darwin
+    goarch:
+      - "386"
```

Comment:
Do we have any reason for the 32bit builds?

Reply:
We supported that also before (it's enabled by default, see [here](https://goreleaser.com/customization/build/#builds)). Not including this would be a breaking change (thus would need a `v2`).

Reply:
Okay, fair 🤷🏼‍♂️